### DISCUSSION TITLE
Best way to update included_apps.txt or excluded_apps.txt?

### DISCUSSION THREAD

**Discussion by eirinn1975** [Romanitho/Winget-AutoUpdate#503] [Q&A]
Hello,
I'm deploying the auto-update scripts to a test device group via Intune. 
Everything's going well, but I wanted to update the list contained in the included_apps.txt file (this is a valid question also for excluded_apps.txt file of course).
What would be the best way to deploy the updated version? If I simply repack the scripts, since the app detection rule is the same, intune will consider the script as already installed.

**Comment by Romanitho** [comment]
you can base your detection method on the modification date of your included/excluded list.
You can also opt for the admx way, and easily edit your list.

**Comment by ja-law** [comment]
I use the parameter -ListPath to point to a web address folder where I can maintain the excluded list text file
